### PR TITLE
Hacking rsync

### PR DESCRIPTION
**Pull request by jelly**
Follow up from https://github.com/cockpit-project/cockpit/pull/18325

### CODE REVIEW

**Comment by jelly** [comment]
Once the text is approved this can go to machines, podman, etc.

File: README.md [hunk 1]
```diff
@@ -51,6 +51,11 @@ the remote hostname.
 
     $ RSYNC=c make watch
 
+When developing against a remote host as a normal user `RSYNC_DEVEL` can be set
```
Comment (martinpitt) [inline README.md:54]:
```suggestion
When developing against a remote host as a normal user, `RSYNC_DEVEL` can be set
```

**Review by martinpitt — CHANGES_REQUESTED** [review]
Thanks! Let's make this a bit easier to read, but LGTM!

File: README.md (hunk 1 above)
Comment (jelly) [reply README.md]: Applied.

**Pull request merged**